src/test/java/com/tikal/hudson/plugins/notification/PhaseTest.java

```diff
@@ -38,10 +38,6 @@ public class PhaseTest {
   @Mock
   private EnvVars environment;
   @Mock
-  private Protocol protocol;
-  @Mock
```

Comment:
Removing the mock here to use Spy later because Since Java 15 abstract enums are declared sealed, which prevents mocking